protocol/xmrtaker/net.go

```diff
@@ -41,10 +42,26 @@ func (a *Instance) initiate(providesAmount common.EtherAmount, receivedAmount co
 	}
 
 	// check user's balance and that they actually have what they will provide
```

Comment:
These `x.Cmp(y)` comparisons take a few seconds for the reader to process. It might be nice if the comment said:
```
// Ensure the user's balance is strictly greater than the amount they will provide
```

Reply:
updated!